js/src/name/publish.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env mocha */
+'use strict'
+
+const each = require('async/each')
+const hat = require('hat')
+
+const { fixtures } = require('./utils')
+const { getDescribe, getIt, expect } = require('../utils/mocha')
+
+module.exports = (createCommon, options) => {
+  const describe = getDescribe(options)
+  const it = getIt(options)
+  const common = createCommon()
+
+  describe('.name.publish', function () {
+    this.timeout(50 * 1000)
```

Comment:
Traditionally we've favored test specific timeouts over global timeouts so that we're alerted if a single test that usually takes a very short amount of time starts to take a long amount of time. Can we please remove this and add increased timeouts to the tests that need them?